Caliptra_WG_Technical_Charter.md

```diff
@@ -58,23 +64,43 @@ The responsibilities of these TAC roles within the Workgroup include:
 - Determining whether any Workgroup open source collaborations should be wound
   down and archived on GitHub.
 
+### Classes of TAC members
+
+Due to the specific nature of the Caliptra project, there are three classes of
+TAC members. All members are expected to fulfill the responsibilities
+outlined above. However, the subject matter of different classes of maintainers
+is scoped to an area of the Caliptra project:
+
+- *RTL* Maintainers: these maintainers provide oversight over all contributions
+  to digital logic and silicon design.
+- *Firmware* Maintainers: these maintainers provide oversight over all
+  contributions to immutable ROM (Read Only Memory) code, First Mutable Code
+  (FMC) and runtime firmware (FW).
+- *Voting* Members: Only the Voting Members cast the votes for all decisions
+  below that require a TAC vote. Where TAC quorum may be needed, only a quorum
+  of Voting Members is required. Voting Members are responsible for membership
+  decisions in the TAC.
+
+All members of the TAC share oversight responsibilities for miscellaneous code
+that does not fall in any of the categories above, and for documentation files.
+
 ## Eligibility
 
-`[The initial TAC members of the Workgroup will be representatives appointed by
-[WORKGROUP FOUNDING ORGANIZATIONS], with one representative per organization.]`
+The initial TAC members of the Caliptra workgroup are listed in the
+[MAINTAINERS](MAINTAINERS.md) file committed in the same commit as this line.
 
 Organizations who have joined the CHIPS Alliance and signed the CHIPS Alliance
 Participation Agreement and CHIPS Alliance Contributor License Agreement are
 eligible to nominate developer representatives to join the Workgroup TAC.
 
-Developer representatives may join the TAC subject to a simple majority approval
-vote by the existing TAC members in attendance of a TAC meeting, provided quorum
-is met, or by electronic vote without a meeting, with a majority vote of all
-voting members of the TAC.
+At all times, a CHIPS Alliance member organization that meets the CLA
```

Comment:
I received feedback that this is unclear. Can define this as a "Caliptra member organization", and then rephrase to:
"At all times, a Caliptra member organization shall declare one employee for each class of TAC member: RTL, Firmware, Voting"

Reply:
Overall really appreciating these clarifying edits, they are great. I think "Caliptra member organization" might be confusing here, though. That makes it sound like there is a membership class within Caliptra itself. These are CHIPS Alliance members who appoint technical committee representatives to the Workgroup. Point taken on wanting to improve clarity of wording but I'm not sure this is the way to rephrase.

Reply:
What about this: although there is no separate membership agreement, etc., if NVIDIA would like a term for referring to organizations that have appointed representatives to the Caliptra Workgroup TAC, would it work to summarize and define as "Caliptra Participating Organizations"? E.g., editing lines 92-94 to "Organizations who have joined the CHIPS Alliance and signed the CHIPS Alliance
Participation Agreement and CHIPS Alliance Contributor License Agreement are
eligible to nominate developer representatives to join the Workgroup TAC; upon acceptance of an employee to the Workgroup TAC, an organization becomes a "Caliptra Participating Organization"." and editing lines 96-97 to "At all times, a Caliptra Participating Organization should have one employee included in each..."

Reply:
sgtm. Varun and team, does this proposal work for you?

Reply:
Yes. Hilary's proposal is good. The choice between "should have one employee" vs "shall" we leave up to you, as long as it's clear on the requirements for the Participating Organizations.